da.yml

```diff
@@ -0,0 +1,317 @@
+  INSTTYPE_STANDARD: "&Standard installation (anbefalet)"
+  INSTTYPE_STANDARD_DESC: "Installer $(^NameDA) for alle brugere på denne maskine."
+  INSTTYPE_PORTABLE: "&Bærbar installation"
+  INSTTYPE_PORTABLE_DESC: "Udpak en bærbar version af $(^NameDA). I bærbar tilstand, er al konfigurationsdata opbevaret i applikationsmappen og intet information er skrevet til registret."
+  EXTRACTING_FILES: "Udpakker filer... %s"
+  INETC_DOWNLOADING: "Henter %s"
+  INETC_CONNECTING: "Forbinder..."
+  INETC_SECOND: "sekunder"
+  INETC_MINUTE: "minutter"
+  INETC_HOUR: "timer"
+  INETC_PROGRESS: "%dkB (%d%%) af %dkB @ %d.%01dkB/s"
+  INETC_REMAINING: " (%d %s%s tilbage)"
+tray:
+  TRAY_OPEN: "Åben &Windhawk"
+  TRAY_LOADED_MODS: "Indlæste &modifikationer"
+  TRAY_TOOLKIT: "&Værktøjskasse"
+  TRAY_EXIT: "Afslut"
+  EXITDLG_TITLE: "Afslut Windhawk"
+  EXITDLG_CONTENT: "Windhawk vil afslutte og alle modifikationer vil blive udlæst. Windhawk vil blive startet automatisk ved næste opstart, hvis indstillingen forneden er markeret."
+  EXITDLG_CHECKBOX_AUTOSTART: "Start ikke ved næste opstart."
+  EXITDLG_BUTTON_EXIT: "Afslut Windhawk"
+  EXITDLG_BUTTON_CANCEL: "Annuller"
+  NOTIFICATION_UPDATE_APP: "En ny version af Windhawk er tilgængelig"
+  NOTIFICATION_UPDATE_APP_MOD: "En ny version af Windhawk er tilgængelig, og en modifikation kan blive opdateret"
+  NOTIFICATION_UPDATE_APP_MODS: "En ny version af Windhawk er tilgængelig, og %d modifikationer kan blive opdateret"
+  NOTIFICATION_UPDATE_MOD: "En Windhawk-modifikation kan opdateres"
+  NOTIFICATION_UPDATE_MODS: "%d modifikationer kan blive opdateret"
+  TRAYICON_TOOLTIP_UPDATE: "En ny version er tilgængelig"
+  TASKDLG_TITLE_LOADED_MODS: "Indlæste modifikationer"
+  TASKDLG_TITLE_TASKS_IN_PROGRESS: "Modifikationsopgaver i gang"
+  TASKDLG_BUTTON_OPEN_APP: "Åben %Windhawk"
+  TASKDLG_COLUMN_MOD: "Modifikation"
+  TASKDLG_COLUMN_PROCESS: "Proces"
+  TASKDLG_COLUMN_PID: "PID"
+  TASKDLG_COLUMN_STATUS: "Status"
+  TASKDLG_STATUS_PENDING: "Venter..."
+  TASKDLG_STATUS_LOADING: "Indlæser..."
+  TASKDLG_STATUS_LOADED: "Indlæst"
+  TASKDLG_STATUS_UNLOADED: "Udlæst"
+  TASKDLG_TASK_INITIALIZING: "Initialiserer..."
+  TASKDLG_TASK_LOADING_SYMBOLS: "Indlæser symboler..."
+  TOOLKITDLG_TITLE: "Windhawk værktøjskasse"
+  TOOLKITDLG_BUTTON_OPEN: "Åben Windhawk"
+  TOOLKITDLG_BUTTON_LOADED_MODS: "Indlæste &modifikationer"
+  TOOLKITDLG_BUTTON_EXIT: "Afslut Windhawk"
+  TOOLKITDLG_BUTTON_SAFE_MODE: "%Sikker tilstand"
+  TOOLKITDLG_BUTTON_CLOSE: "&Luk værktøjskasse"
+  SAFE_MODE_TITLE: "Genåbn i sikker tilstand"
+  SAFE_MODE_TEXT: "Windhawk vil blive genåbnet med al kodeinjections-funktionaliteten slået fra."
+  TASKDLG_PROCESS_SUSPENDED: "(Suspenderet)"
+  SAFE_MODE_DETECTED_TITLE: "Sikker tilstand"
+  SAFE_MODE_DETECTED_TEXT: "Windows kører i sikker tilstand. Vil du slå sikker tilstand til i Windhawk? mode in Windhawk? Al kodeinjections-funktionalitet vil blive slået fra."
```

Comment:
Please add and translate this newly added string (after `SAFE_MODE_DETECTED_TEXT`):
https://github.com/ramensoftware/windhawk-translate/blob/17dc5116a4a1e91631c03d4dfe2e7d9a17bcafcf/en.yml#L76